test/services/offenderService.test.js

```diff
@@ -3550,3 +3639,204 @@ describe('statusTextDisplay', () => {
+      dbRecord: {
+        status: Status.APPROVED.name,
+      },
+      releaseDate: mockTodaySubtract(1),
+      expected: false,
+    },
+    {
+      description: 'should return false if status is not supervisor back and release date is in the future',
+      dbRecord: {
+        status: Status.APPROVED.name,
+      },
+      releaseDate: moment().add(1, 'week'),
+      expected: false,
+    },
+    {
+      description: 'should return false if status is not supervisor back and release date is today',
+      dbRecord: {
+        status: Status.APPROVED.name,
+      },
+      releaseDate: moment(),
+      expected: false,
+    },
+    {
+      description: 'should return false if status is supervisor back and release date is in the past',
+      dbRecord: {
+        status: Status.SUPERVISOR_BACK.name,
+      },
+      releaseDate: mockTodaySubtract(37),
+      expected: false,
+    },
+    {
+      description: 'should return true if status is supervisor back and release date is in the future',
+      dbRecord: {
+        status: Status.SUPERVISOR_BACK.name,
+      },
+      releaseDate: moment().add(1, 'month'),
+      expected: true,
+    },
+    {
+      description: 'should return true if status is supervisor back and release date is null',
+      dbRecord: {
+        status: Status.SUPERVISOR_BACK.name,
+      },
+      releaseDate: null,
+      expected: true,
+    },
+    {
+      description: 'should return true if status is supervisor back and release date is today',
+      dbRecord: {
+        status: Status.SUPERVISOR_BACK.name,
+      },
+      releaseDate: moment(),
+      expected: true,
+    },
+    {
+      description: 'should return false if status is invalid and release date is in the past',
+      dbRecord: {
+        status: 'invalid-status',
+      },
+      releaseDate: mockTodaySubtract(44),
+      expected: false,
+    },
+    {
+      description: 'should return true if status is supervisor back and release date is not a valid date',
+      dbRecord: {
+        status: Status.SUPERVISOR_BACK.name,
+      },
+      releaseDate: 'invalid-date',
+      expected: true,
+    },
+    {
+      description: 'should return true if status is supervisor back and release date is null',
+      dbRecord: {
+        status: Status.SUPERVISOR_BACK.name,
+      },
+      releaseDate: null,
+      expected: true,
+    },
+    {
+      description: 'should return false if status is missing and release date is in the past',
+      dbRecord: {},
+      releaseDate: mockTodaySubtract(2),
+      expected: false,
+    },
+    {
+      description: 'should return true if status is supervisor back and release date is missing',
+      dbRecord: {
+        status: Status.SUPERVISOR_BACK.name,
+      },
+      releaseDate: null,
+      expected: true,
+    },
+  ]
+
+  testCases.forEach(({ description, dbRecord, releaseDate, expected }) => {
+    it(description, () => {
```

Comment:
appreciate this does not use the `each` approach - this is a copy from the previous PR where these tests were passing and I do not have time to refactor right now